deploy.sh

```diff
@@ -0,0 +1,403 @@
+#!/bin/bash
+
+# Exit on error
+set -e
```

Comment:
@hrxi  says we should do this.
He also says the last line might not work on ~~trash~~ Mac.

```suggestion
set -eu
set -o pipefail
```